iree/compiler/Dialect/Flow/Transforms/ConvertLinalgMatmulToMmt4D.cpp

```diff
@@ -385,7 +425,12 @@ class ConvertLinalgMatmulToMmt4DPass final
 
   LogicalResult initializeOptions(StringRef options) override {
     if (failed(Pass::initializeOptions(options))) return failure();
-    return ParseCustomKernelsTargetInfo(arch, features, target_info);
+    if (failed(ParseCustomKernelsTargetInfo(arch, features, target_info))) {
+      llvm::errs() << "Bad options `" << options << "` for pass `"
```

Comment:
I think we generally don't want to just write to stderr. Does the pass infra not give a helpful error with bad options? That seems like something to fix at a higher level.

Reply:
Yeah I think it should be printing this out here: https://github.com/llvm/llvm-project/blob/bfc6fbfb65f6d490e6a1ba4eb6c734d2b4494dd1/mlir/lib/Pass/PassRegistry.cpp#L523

Reply:
ok, reverted (FWIW I wasn't able to see an error, but maybe i don't have the right command lines to enable that output).